spec/helper_spec.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+TestStruct = Struct.new(:name, :other_param)
+
+RSpec.describe Surrealist::Helper do
+  describe 'serializing  a single struct' do
+    let(:person) { TestStruct.new('John', 'Dow') }
+
+    it 'raise surrealize instead surrealize_collection with struct' do
```

Comment:
everything seems good, but the example message is quite misleading: nothing is raised, we are just asserting that `SurrealistHelper.collection?` returns false for a single struct
I suggest something like `specify 'a struct is not treated as a collection' do ...`

Reply:
fixed